packages/opentelemetry-instrumentation/README.md

```diff
@@ -1,6 +1,5 @@
 # OpenTelemetry Instrumentation for web and node
 
-[![Gitter chat][gitter-image]][gitter-url]
```

Comment:
missing discussion section

Reply:
This is the badge list. There is no discussion badge

Reply:
you have changed gitter url tto discussion and you are not using this, in all other readme you have
```
For help or feedback on this project, join us in [GitHub Discussions][discussions-url]
```
but not here

Reply:
Ah ok

Reply:
ok but are you going to add this 
```
For help or feedback on this project, join us in [GitHub Discussions][discussions-url]
```
or remove the unused url ?

Reply:
it's there https://github.com/open-telemetry/opentelemetry-js/pull/1866/files#diff-78489be54819b6a4a6fe33c1e4158ee8c292904bcc9d34a76e150e25b4628109R273